src/subscription_manager/managercli.py

```diff
@@ -1706,7 +1711,8 @@ def _do_command(self):
                 )
         except (connection.RestlibException, exceptions.ServiceError) as re:
             log.exception(re)
-            system_exit(os.EX_SOFTWARE, re)
+            mapped_message: str = ExceptionMapper().get_message(err)
```

Comment:
I think this will need the same fix that was done in one of the commits of #2770. No need to backport that PR as a whole -- I'd simply fix it directly in the same commit, and mention in the commit message that it contains an extra fix from 3095f59a8b9a14a42e4eecb12b7af29f44e6f2b7.